purchase_exception/migrations/12.0.1.0.1/noupdate_changes.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+  <record id="ir_cron_test_po_order_except" model="ir.cron" forcecreate="True">
+    <field name="code">model.test_all_draft_orders()</field>
+  </record>
+  <record id="po_excep_no_email" model="exception.rule">
+<!--    <field name="rule_group"/>-->
+    <field name="code">if not self.partner_id.email:
+            failed=True</field>
+  </record>
+  <record id="pol_excep_qty_check" model="exception.rule">
+<!--    <field name="rule_group"/>-->
```

Comment:
Cleanup unused code

Reply:
It's better to not delete in this case. If in the future we check again the noupdate changes from v11 and v12 versions, then this way we will not be tempted to add the commented code.